stock_putaway_product/wizard/stock_change_product_qty.py

```diff
@@ -21,9 +22,8 @@ def default_get(self, fields):
             putaway_ids = self.env['stock.product.putaway.strategy'].search([
```

Comment:
Rename this to `putaway` (without `s` nor `ids`), as it reflects better the content of the variable.